app/api/domains/osu.py

```diff
@@ -1166,10 +1165,22 @@ async def getScores(
     if map_md5 in app.state.cache.needs_update:
         return b"1|false"
 
-    mods = Mods(mods_arg)
-    mode = GameMode.from_params(mode_vn, mods)
+    mods = mods_arg
```

Comment:
do we really need to redefine these, we could just change the names in the function args no?